commit dd00b11daf2b869c24f22d6d21932507b3f2413a (refs/changes/42/942/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-11T10:44:03-08:00 (2 years, 9 months ago)
    
    joyent/manta-thoth#144 bump dependency versions and release 1.0.0
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: Bryan Cantrill <bryan@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>
    Approved by: Bryan Cantrill <bryan@joyent.com>

diff --git a/bin/thoth b/bin/thoth
index 32ebe6b..13207a2 100755
--- a/bin/thoth
+++ b/bin/thoth
@@ -10,7 +10,6 @@ var mod_fs = require('fs');
 var util = require('util');
 var mod_crypto = require('crypto');
 var mod_bunyan = require('bunyan');
-var mod_restify = require('restify');
 var mod_manta = require('manta');
 var mod_zlib = require('zlib');
 var mod_child = require('child_process');
@@ -3455,8 +3454,7 @@ var connect = function ()
 	opts.log = mod_bunyan.createLogger({
 		name: mod_path.basename(process.argv[1]),
 		level: (process.env.LOG_LEVEL || 'info'),
-		stream: process.stderr,
-		serializers: mod_restify.bunyan.serializers
+		stream: process.stderr
 	});
 
 	return (mod_manta.createBinClient(opts));
diff --git a/package.json b/package.json
index d872586..3fcb1fc 100644
--- a/package.json
+++ b/package.json
@@ -1,15 +1,14 @@
 {
 	"name": "manta-thoth",
 	"description": "Manta-based system for core and crash dump analysis",
-	"version": "0.2.2",
+	"version": "1.0.0",
 	"author": "Joyent (joyent.com)",
 	"dependencies": {
-		"manta": "1.6.0",
+		"manta": "^4.0.0",
 		"ctype": ">=0.5.2",
 		"sprintf": "0.1.1",
-		"bunyan": "0.21.3",
-		"jsprim": "^0.8.0",
-		"restify": "2.5.0",
+		"bunyan": "^1.8.5",
+		"jsprim": "^1.3.1",
 		"rethinkdb": "2.0.0",
 		"readable-stream": ">=1.1.9"
 	},
@@ -19,6 +18,5 @@
 	"repository": {
 		"type" : "git",
 		"url" : "http://github.com/joyent/manta-thoth.git"
-	},
-	"engines": { "node": ">=0.8" }
+	}
 }
